src/components/ec-navigation-arrows/ec-navigation-arrows.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div
+    class="ec-navigation-arrows"
+    data-test="ec-navigation-arrows"
+  >
+    <button
+      type="button"
+      :class="leftButtonClasses"
+      :disabled="isLeftArrowDisabled"
+      class="tw-mr-12"
+      data-test="ec-navigation-arrows__left"
+      @click="emit(NavigationArrowsEvent.LEFT_ARROW_CLICK)"
+    >
+      <ec-icon :size="24" :name="IconName.SimpleChevronLeft" /></button>
+    <button
+      type="button"
+      :class="rightButtonClasses"
+      :disabled="isRightArrowDisabled"
+      data-test="ec-navigation-arrows__right"
+      @click="emit(NavigationArrowsEvent.RIGHT_ARROW_CLICK)"
+    >
+      <ec-icon :size="24" :name="IconName.SimpleChevronRight" /></button>
+  </div>
+</template>
+
+<script setup lang="ts">
+
+import { computed } from 'vue';
+
+import EcIcon from '../ec-icon/ec-icon.vue';
+import { IconName } from '../ec-icon/types';
+import { NavigationArrowsEvent } from './types';
+
+const emit = defineEmits<{
+  'right-arrow-click': []
+  'left-arrow-click': [],
+}>();
+
+interface NavigationArrowsProps {
+  isRightArrowDisabled?: boolean
+  isLeftArrowDisabled?: boolean
+}
+
+const props = defineProps<NavigationArrowsProps>();
+
+const leftButtonClasses = computed(() => getButtonClasses(props.isLeftArrowDisabled));
+
+const rightButtonClasses = computed(() => getButtonClasses(props.isRightArrowDisabled));
+
+function getButtonClasses(isDisabled: boolean) {
+  return {
+    'ec-navigation-arrows__button': true,
+    'ec-navigation-arrows__button--disabled': isDisabled,
+  };
+}
+</script>
+
+<style>
+.ec-navigation-arrows {
+  @apply tw-text-gray-4 tw-h-24;
+
+  &__button {
+    @apply tw-h-24 tw-cursor-pointer;
+    @apply tw-border-0 tw-p-0 tw-bg-transparent tw-fill-current;
+
+    &:hover {
```

Comment:
We need to apply this for `:focus` too for a11y.

Reply:
The problem with this :focus is that if you focus using keyboard, then hover cursor and move cursor away - focus is still applied. Also if you click using mouse on button, focus is set and even if cursor is not over the button it renders as 
hovered.
![image](https://github.com/Ebury/chameleon/assets/5939240/7cc765a4-92c3-4c9b-ad8c-e4a05351ca22)

Not sure this is the behaviour we want to have. We could introduce js code to handle it, but is it worth to support this focus color change as we already have outline on selected button?
![image](https://github.com/Ebury/chameleon/assets/5939240/32097496-ca3f-4bc6-89d5-af2dbc11d830)

Reply:
The user must be able to tell which element is focused right now. We must have ability to navigate through clickable components with just the keyboard and it must be clear where the focus currently is. I don't mind that the element is still highlighted after the mouseout. The ugly black outline of the Chrome focus should not be used. Basically, check the ec-btn, this can be rendered as `<button>` and `<a>`, they have focus and hover as well. We also have `Elements` section in the storybook for checking bare element styles, check the anchors and how they behave when focused and hovered.